From 40d90725f39101f2f548a7279cb4fb7911e9cb54 Mon Sep 17 00:00:00 2001
From: He Sheng <hesheng@wxiat.com>
Date: Tue, 7 Jun 2022 17:41:54 +0800
Subject: [PATCH 2991/4585] [Backport] sw64: clean up useless #if 0 and #if 1

Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

Reference: https://gitee.com/openeuler/kernel/commit/d55ea04bddf236a4dcc3fe2b42b1fbaf5af5420b

--------------------------------

They make no sense and should be removed.

Signed-off-by: He Sheng <hesheng@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Acked-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>

Signed-off-by: Wang Hai <wanghai38@huawei.com>
---
 arch/sw_64/include/uapi/asm/auxvec.h | 2 --
 arch/sw_64/kernel/module.c           | 4 ----
 2 files changed, 6 deletions(-)

diff --git a/arch/sw_64/include/uapi/asm/auxvec.h b/arch/sw_64/include/uapi/asm/auxvec.h
index 59854f3ac501..5b554a593fc0 100644
--- a/arch/sw_64/include/uapi/asm/auxvec.h
+++ b/arch/sw_64/include/uapi/asm/auxvec.h
@@ -3,10 +3,8 @@
 #define _UAPI_ASM_SW64_AUXVEC_H
 
 /* Reserve these numbers for any future use of a VDSO.  */
-#if 1
 #define AT_SYSINFO		32
 #define AT_SYSINFO_EHDR		33
-#endif
 
 /*
  * More complete cache descriptions than AT_[DIU]CACHEBSIZE.  If the
diff --git a/arch/sw_64/kernel/module.c b/arch/sw_64/kernel/module.c
index c75d8a2e4309..d2041afba7bb 100644
--- a/arch/sw_64/kernel/module.c
+++ b/arch/sw_64/kernel/module.c
@@ -7,11 +7,7 @@
 #include <linux/kernel.h>
 #include <linux/slab.h>
 
-#if 0
-#define DEBUGP printk
-#else
 #define DEBUGP(fmt...)
-#endif
 
 /* Allocate the GOT at the end of the core sections.  */
 
-- 
2.17.1

